package.json

```diff
@@ -91,24 +91,23 @@
   },
   "types": "./build/lib/driver.d.ts",
   "devDependencies": {
-    "@appium/eslint-config-appium": "8.0.5",
-    "@appium/types": "0.17.0",
-    "@semantic-release/changelog": "6.0.3",
-    "@semantic-release/git": "10.0.1",
-    "@tsconfig/node14": "14.1.2",
-    "@types/bluebird": "3.5.42",
-    "@types/mocha": "10.0.9",
-    "@types/node": "20.16.15",
-    "asyncbox": "3.0.0",
+    "@appium/eslint-config-appium": "^8.0.5",
+    "@appium/tsconfig": "^0.x",
+    "@appium/types": "^0.x",
+    "@semantic-release/changelog": "^6.0.3",
+    "@semantic-release/git": "^10.0.1",
+    "@types/bluebird": "^3.5.42",
+    "@types/mocha": "^10.0.9",
+    "@types/node": "^20.16.15",
+    "asyncbox": "^3.0.0",
     "chai": "^5.1.1",
-    "chai-as-promised": "8.0.0",
-    "conventional-changelog-conventionalcommits": "8.0.0",
-    "eslint": "8.57.1",
-    "mocha": "10.7.3",
-    "rimraf": "5.0.10",
-    "semantic-release": "24.1.3",
-    "ts-node": "10.9.2",
-    "typescript": "5.6.3",
+    "chai-as-promised": "^8.0.0",
+    "conventional-changelog-conventionalcommits": "^8.0.0",
+    "mocha": "^10.7.3",
+    "rimraf": "^5.0.10",
+    "semantic-release": "^24.1.3",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.6.3",
     "webdriverio": "9.0.9"
```

Comment:
should wdio version be locked?

Reply:
Yea, to keep CI succeeding for now. I need to spend time to investigate this webdriver version related test failure, which version started having an issue for example. So as this PR, I keep this wdio version.
https://github.com/appium/appium-chromium-driver/pull/348